web/src/main/java/com/codehustle/ems/service/impl/EmployeeServiceImpl.java

```diff
@@ -34,4 +35,23 @@ public void addEmployee(Employee employee) throws ServiceException {
             throw new ServiceException(e.getMessage());
         }
     }
+
+    public void deleteEmployee(Employee employee) throws ServiceException
```

Comment:
In controller you are sending empId,
in service you are receiving employee? how?